package.json

```diff
@@ -62,7 +62,10 @@
     "vue": "^2.4.2"
   },
   "dependencies": {
-    "bootstrap": "^4.0.0-beta"
+    "bootstrap": "^4.0.0-beta",
```

Comment:
The only issue about having Bootstrap placed here as a dependancy is that Bootstrap Vue will use the official Bootstrap v4 CSS as is.

If one wants to use a customized Bootstrap CSS in their project, Bootstrap-Vue may not use it their custom compiled CSS.